nixosModules/homestakeros/options.nix

```diff
@@ -1,6 +1,19 @@
 { lib
+, cfg
 , ...
-}: {
+}:
+let
+  # Try to get the first mount's path, fallback to null
+  firstMountPath =
+    let
+      mountNames = lib.attrNames (lib.filterAttrs (name: mount: mount.enable) cfg.mounts);
+      firstMountName = lib.optional (mountNames != [ ]) (builtins.head mountNames);
+    in
+    if firstMountName != [ ]
+    then "${cfg.mounts.${builtins.head firstMountName}.where}"
+    else null;
```

Comment:
Because when `mountNames` is always an array, you can always call array functions like head on it.